App add routes under app

TurpaultM opened the pull request:
This pull request introduces a set of Click & Collect endpoints and CRUD operations for ordonnances. These endpoints support sending orders, validating/refusing them, managing temporary images, and creating or deleting ordonnances.

**Click & Collect Endpoints**

* Added endpoints to send orders, fetch commands, get requests, validate or refuse orders, check status, and manage temporary images. (`backend/routes/clickcollect/*.py`)

**Ordonnances Endpoints**

* Added endpoints to create, fetch, and delete ordonnances for users. (`backend/routes/ordonnances/*.py`)

**Blueprint Registration**

* Registered all new Click & Collect and ordonnances blueprints in the main application setup to enable their routes. (`backend/app.py`)

---

**Tuto on how to test it**

*Start*

1. Do this in your terminal:

```bash
python launch.py --down && python launch.py --verif --back --no-cache-back
```

*Ordonnances routes*

2. `Create Ordonnance`: Use `POST /ordonnances/create` to create a new ordonnance for a user.
3. . `Get User Ordonnances`: Use `GET /ordonnances?user_id=` to fetch all ordonnances for a user.
4. `Delete Ordonnance`: Use `DEL /ordonnances/delete` to delete an ordonnance by ID.

*Temporary Image routes*

5. `Upload Temp Image`: Use `POST /clickcollect/upload_temp_image` to upload a temporary image.
6. `Get Temp Images by User`: Use `GET /clickcollect/temp_image?user_id=` to fetch temp images for a user.
7. `Get Temp Image by ID`: Use `GET /clickcollect/temp_image?image_id=` to fetch a specific temp image.
8. `Delete Temp Images by User`: Use `DEL /clickcollect/temp_image?user_id=` to delete all temp images for a user.
9. `Delete Temp Image by ID`: Use `DEL /clickcollect/temp_image?image_id=` to delete a specific temp image.

*Click & Collect routes*

10. `Send Order`: Use `POST /clickcollect/send` to create a new Click & Collect order.
11. `Get Commands`: Use `GET /clickcollect/commands?user_id=` to fetch all commands for a user.
12. `Get Requests`: Use `GET /clickcollect/requests` to fetch all pending requests.
13. . `Validate Order`: Use `POST /clickcollect/validate` to validate a pending order.
14. . `Refuse Order`: Use `POST /clickcollect/refuse` to refuse a pending order.
15. . `Check Status`: Use `GET /clickcollect/status?order_id=` to check the status of an order.

TurpaultM committed:
Merge branch 'APP_link-app-with-new-back' of github.com:Jessika63/PharmaXcess into APP_link-app-with-new-back

TurpaultM committed:
Merge branch 'APP_link-app-with-new-back' of github.com:Jessika63/PharmaXcess into APP_link-app-with-new-back

TurpaultM committed:
ADD: routes for under app and add prescription add/get/delete routes

TurpaultM committed:
ADD: routes for under app and add prescription add/get/delete routes

TurpaultM committed:
ADD: routes for under app and add prescription add/get/delete routes

TurpaultM committed:
ADD: routes for under app and add prescription add/get/delete routes

ErwannL reviewed (changes requested):
as discuss on discord, need step of testing

ErwannL reviewed (changes requested):
As discuted on discord 
see [here](https://discord.com/channels/1256399262934372386/1436049206170091642/1436412458783866940)
need changes

TurpaultM committed:
UPDATE and ADD new routes for prescription, under app and login

ErwannL committed:
Update app DB dump references to latest date

Changed references to the app database dump from 16_10_2025 to 04_12_2025 in config files to use the most recent backup.
